demo/linux_app/Dockerfile_linux_app

```diff
@@ -0,0 +1,37 @@
+# BUILDER
+FROM ubuntu:22.04
+
+# build ARGs
+ARG _LINUX_GIT_REPO='https://github.com/torvalds/linux.git'
+ENV LINUX_GIT_REPO=$_LINUX_GIT_REPO
+ARG _LINUX_GIT_BRANCH="v6.6"
```

Comment:
6.6.44

Reply:
Couldn't locate this tag in the upstream.

Reply:
Use greg's so we can narrow in on a stable bugfix minor rev.

https://github.com/gregkh/linux.git

v6.6.47